Documentation/update-codeowners.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source_dir="$(cd "${script_dir}/.." && pwd)"
+
+echo "<!-- This file was autogenerated via Documentation/update-codeowners.sh, do not edit manually-->" \
```

Comment:
rST comment?

```suggestion
echo ".. This file was autogenerated via Documentation/update-codeowners.sh, do not edit manually" \
```

Reply:
:+1: This is following the same format as is used in `cmdref` docs, but maybe that's a sign that the cmdref docs should be updated. I'll fix this up for the script here.

Reply:
I think it would not render properly if someone tried to generate Latex or an epub, for example. The RST comment syntax should be converted appropriately, whatever the target.